commit 95bce8afd6973b00232f3cb7cc286d3994a07a13
Author: Andrzej Telszewski <atelszewski@gmail.com>
Date:   Sat Oct 24 12:51:20 2020 +0200

    Change how git-store-meta is called from Git hooks
    
    Call `git-store-meta` instead of `git-store-meta.pl`.

diff --git a/git-store-meta.pl b/git-store-meta.pl
index ddc0024..b0d0d37 100755
--- a/git-store-meta.pl
+++ b/git-store-meta.pl
@@ -600,7 +600,7 @@ sub install_hooks {
     my $mask = umask; if (!defined($mask)) { $mask = 0022; }
     my $mode = 0777 & ~$mask;
     my $t;
-    my $s = escapeshellarg($GIT_STORE_META_APP . ".pl");
+    my $s = escapeshellarg($GIT_STORE_META_APP);
     my $f = defined($argv{'target'}) ? " -t " . escapeshellarg($argv{'target'}) : "";
     my $f2 = escapeshellarg(defined($argv{'target'}) ? $argv{'target'} : $GIT_STORE_META_FILENAME);
 
